gpkit/model.py

```diff
@@ -430,7 +430,8 @@ def formProgram(self, programType, signomials, subs, verbosity=2):
         for s in signomials:
             _, exps, cs, _ = substitution(s.varlocs, s.varkeys,
                                           s.exps, s.cs, subs)
-            if any((mag(c) != 0 for c in cs)):
+            notnan = ~np.isnan(cs)
+            if np.any(notnan) and np.any(cs[notnan] != 0):
```

Comment:
is this the implementation of "drop posynomials with only nan coefficients"?

I think that deserves an inline comment.